compute/computer_setup/install_magaox_rpi.rst

```diff
@@ -0,0 +1,103 @@
+Installing MagAO-X on a Raspberry Pi
+====================================
+
+
+Configuring the Raspberry Pi
+----------------------------
+
+Prepare the MicroSD card
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first step is to download the Raspberry Pi Imager from the `Raspberry Pi Website <https://www.raspberrypi.com/software/>`__. The MicroSD card can then 
+be inserted into your computer. Now open the Raspberry Imager and select the Raspberry Pi Device you have, the operating system (OS) (Ubuntu server 24.04 LTS), 
+and the storage (your MicroSD card). To access the OS customization, use ``ctrl+shift+x``. This is where you can set hostname, username password, 
+etc. It is crucial that you 'Enable SSH' in the OS customization. Now click 'next' and 'write' to install the OS on the Micro SD card. 
```

Comment:
the crucial Enable SSH step could be called out as a "node" so it's really clear